tests/integration/suite/test_kontainer_drivers.py

```diff
@@ -289,7 +289,7 @@ def test_kontainer_driver_links(admin_mc):
     images = get_images(url, token)
     assert "hyperkube" in images
     assert "rke-tools" in images
-    assert "kubelet-pause" in images
+    assert "mirrored-pause" in images
```

Comment:
Note to Reviewers:
  1.23 and 1.24 uses `rancher/mirrored-pause` instead of  `rancher/kubelet-pause`